mintpy/ifgram_inversion.py

```diff
@@ -191,7 +191,7 @@ def cmd_line_parse(iargs=None):
             inps.outfile = ['timeseriesRg.h5', 'residualInvRg.h5', 'numInvOffset.h5']
 
         elif inps.obsDatasetName.startswith('ion'):
-            inps.outfile = ['timeseriesIon.h5', 'temporalCoherenceIon.h5', 'numInvIon.h5']
+            inps.outfile = ['Ion.h5', 'temporalCoherenceIon.h5', 'numInvIon.h5']
```

Comment:
Because the solid earth tide correction time-series file is named `SET.h5`. Troposphere model correction time-series file is named like `ERA5.h5`. So that is why I did this. 

But keeping it consistent with the current naming style of network inversion also makes sense.

Reply:
This now reverts to the original naming.